fe/fe-core/src/main/java/org/apache/doris/persist/OperationType.java

```diff
@@ -182,4 +182,7 @@
     // resource 276~290
     public static final short OP_CREATE_RESOURCE = 276;
     public static final short OP_DROP_RESOURCE = 277;
+
+    // alter external table
+    public static final short OP_REFRESH_EXTERNAL_TABLE_SCHEMA = 280;
```

Comment:
just puzzled with the `refresh`?

Reply:
OK，I will change the name of this operationType